src/test/resources/testing-editor.md

```diff
@@ -134,10 +139,15 @@
 **Error notification appears when pasting invalid content**
 1. "Edit..." resource
 2. paste invalid yaml
+Bugs:
+1. some weird error message
+2. paste invalid yaml -> close editor -> open same resource -> no invalid string (file was not saved)
 
 **Error notification appears on startup**
 1. have editor with invalid content
 2. restart IJ
+Bugs:
+1. after restart file does not contains the invalid string
```

Comment:
you're referring to the usecase where you had the "weird error message", right? If true, then this is expected. Parsing didn't happen, content thus wasn't saved.